scripts/assets/galoy-API_ssl.conf

```diff
@@ -0,0 +1,19 @@
+## galoy-admin-API_ssl.conf
```

Comment:
I think we're using only lowercase in our repo for filename to avoid issues with the fact not all OS are case sensitives

Reply:
fixed in